src/components/PopularArticles.jsx

```diff
@@ -0,0 +1,38 @@
+export const PopularArticles = ({ articles }) => {
+  const popularArticles = articles
+    .filter((article) => article.votes > 3)
+    .slice(1, 4);
+
```

Comment:
This is an interesting way of getting the most popular articles. I would recommend eventually adding an endpoint to your BE project api that does this for you so this can be handled by a query instead.